keras_cv/models/backbones/csp_darknet/csp_darknet_backbone_presets.py

```diff
@@ -26,15 +26,7 @@
             "official_name": "CSPDarkNet",
             "path": "csp_darknet",
         },
-        "class_name": "keras_cv>CSPDarkNetBackbone",
-        "config": {
-            "stackwise_channels": [48, 96, 192, 384],
-            "stackwise_depth": [1, 3, 3, 1],
-            "include_rescaling": True,
-            "use_depthwise": False,
-            "input_shape": (None, None, 3),
-            "input_tensor": None,
-        },
+        "kaggle_handle": "kaggle://keras/cspdarknet/csp_darknet_tiny/1",
```

Comment:
NIT: Why name this `kaggle_handle`, when you can also give `gs://` for GCS.  Why not have it as generic name so we can also extend it to future like `hf://`, vertext AI endpoint or model garden?

Reply:
This is intended to remain with a Kaggle source of truth. The use of gs:// was temporary for testing while we transition to this, and we intend on hosting all our NLP models on Kaggle.

Reply:
Yeah, extending our hosting options to `gs://`, `s3://`, `hf://` sounds great, but we should keep all of our Keras team models in one place--Kaggle models. Anything else will make for a fractured experience (i.e. with offline use in Kaggle notebooks).